src/app/DTT/upload/dropzone.js

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react'
+import { useDropzone } from 'react-dropzone'
+import { Box, Flex, Text, Input, Button, Center, Image } from '@chakra-ui/react'
+import { isEmpty, map } from 'ramda'
+
+const DropZone = ({ handleSave, closeDropzone, maxSize = '5000000' }) => {
+  const [files, setFiles] = useState([])
+
+  const { getRootProps, getInputProps } = useDropzone({
+    accept: 'image/*',
+    maxSize: maxSize,
+    onDrop: (acceptedFiles, rejectedFiles) => {
+      setFiles(
+        acceptedFiles.map(file =>
+          Object.assign(file, {
+            preview: URL.createObjectURL(file),
+          }),
+        ),
+        rejectedFiles.map(file => alert('There was an error, try again!')),
+      )
+    },
+  })
+
+  const preview = map(file => (
+    <Box w="200px" h="200px" key={file.name}>
+      <Image src={file.preview} alt="drag and drop" boxSize="200px" objectFit="cover" />
+    </Box>
+  ))(files)
+
+  useEffect(
+    () => () => {
+      files.forEach(file => URL.revokeObjectURL(file.preview))
+    },
+    [files],
+  )
+
+  return (
+    <Flex w="100%" p={4} direction="column" bg="white">
+      <Text color="black">{`Upload File`}</Text>
+      <Box {...getRootProps()} p="10" my="4" border="1px black dashed">
+        <Center>
+          <Text color="black">{`Drag 'n' drop some files here, or click to select files`}</Text>
```

Comment:
'n' - love it 😆

Reply:
I don't know if you are being sarcastic so I left it. 😅